index.js

```diff
@@ -143,8 +161,8 @@ Use the getReviewsByRating function below to do the following:
   ]
 */
 
- function getReviewByRating(/* code here */) {
-    /* code here */
+ function getReviewByRating(array1, rating) {
+   return array1.filter(x => x.rating === rating)
   }
```

Comment:
Filter the rating values that are within a range (if 4 is passed in, include all ratings that are between 4 and 5, not inclusive of 5)